pages/api/p/[productId].js

```diff
@@ -30,6 +30,12 @@ export default async function fetchProduct(req, res) {
   if (color) {
     const data = await getPageData(productId)
     data.carousel = { index: 0 }
+    // A price for the fetched product variant would be included in
+    // the response, but for demo purposes only, we are setting the
+    // price based on the color name.
+    const mockPrice = color.split``.reduce((s, e) => s + e.charCodeAt(), 0) / 100
```

Comment:
Size not needed here? Also would be nice if master and commercial logic would look the same, less conflicts